apps/video/index.html

```diff
@@ -33,9 +33,11 @@
 
     <!-- Web Components -->
     <script defer src="shared/elements/config.js"></script>
+    <script defer src="shared/js/component_utils.js"></script>
```

Comment:
Nit (unrelated to this patch): It seems there is some attempt to organize the shared js so it might have been worth it to add more organization by putting utilities supporting web-components in their own sub-directory.